spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/rdd/CassandraRDD.scala

```diff
@@ -105,6 +105,10 @@ class CassandraRDD[R] private[connector] (
     copy(readConf = readConf)
   }
 
+  /** Produces the empty CassandraRDD which has the same signature and properties, but it does not
+    * perform any validation and it does not even try to return any rows. */
+  def toEmptyCassandraRDD = new EmptyCassandraRDD[R](sc, connector, keyspaceName, tableName)
```

Comment:
I decided to remove `EmptyCassandraRDD`.

However - `makeRDD` is a method of `SparkContext`, isn't it? If we decided to do such changes, we should rename `cassandraTable` to `makeCassandraRDD` and add `makeEmptyCassandraRDD`. I'm not convinced for this change in this ticket.